### ISSUE TITLE
Minor bugs on the Research 7 Campaign

### ISSUE THREAD

**Issue by noooxz** [ReactiveDrop/reactivedrop_public_src#631]
Hello developers!

I would like to inform you about two small shortcomings in the campaign Research 7.

The first error is on the "Research 7" map at the very end. The essence of the bug is that the larvae fall under the map and continue to follow the players.
![20230802122036_1](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/136397097/bd4c7083-0588-437c-a9b5-bb9fc9d7f3be)
I also attach it in video format:
https://youtu.be/dTaK0I6s4Zc

The second error is located on the "Illyn Forest" map.

The bottom line is that you can safely walk through the biomass without getting damaged. 
![изображение](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/136397097/8494637b-8f48-4745-95a8-ed1517a9401a)
I also attach it in video format:
https://youtu.be/wNLB8WFJNgM
